docs/breaking-changes.md

```diff
@@ -12,8 +12,73 @@ This document uses the following convention to categorize breaking changes:
 - **Deprecated:** An API was marked as deprecated. The API will continue to function, but will emit a deprecation warning, and will be removed in a future release.
 - **Removed:** An API or feature was removed, and is no longer supported by Electron.
 
+## Planned Breaking API Changes (12.0)
+
+### Removed: `crashReporter` methods in the renderer process
+
+The following `crashReporter` methods are no longer available in the renderer
+process:
+
+- `crashReporter.start`
+- `crashReporter.getLastCrashReport`
+- `crashReporter.getUploadedReports`
+- `crashReporter.getUploadToServer`
+- `crashReporter.setUploadToServer`
+- `crashReporter.getCrashesDirectory`
+
+They should be called only from the main process.
+
+See [#23265](https://github.com/electron/electron/pull/23265) for more details.
+
+## Planned Breaking API Changes (11.0)
+
 ## Planned Breaking API Changes (10.0)
 
+### Deprecated: `companyName` argument to `crashReporter.start()`
+
+The `companyName` argument to `crashReporter.start()`, which was previously
+required, is now optional, and further, is deprecated. To get the same
+behavior in a non-deprecated way, you can pass a `companyName` value in
+`globalExtra`.
+
+```js
+// Deprecated in Electron 10
+crashReporter.start({ companyName: 'Umbrella Corporation' })
+// Replace with
+crashReporter.start({ globalExtra: { _companyName: 'Umbrella Corporation' } })
```

Comment:
This seems super unintuitive and a worse API, is there a reason we _have_ to deprecate / remove the `companyName` thing here, not sure I get the motivation for this change.

Reply:
i disagree that it's unintuitive or worse.

1. Why should the one key `_companyName` be treated specially? None of the underlying logic treats this value specially.
2. `globalExtra` is a more general API and allows you to set arbitrary keys/values to be uploaded with crashes from any process.
3. This makes it clearer what is actually being done with that value.

Reply:
I may be under the misimpression that `_companyName` is a _special_ extra that crashpad handlers treat differently.  i.e. backtrace / sentry explicitly look for that extra vs others which are just "extra keys".  Without `_companyName` being documented no one will know it exists 🤔

Reply:
```
$ rg '\b_companyName\b' ../third_party/crashpad/crashpad ../components/crash
$
```

doubt it.

Reply:
... unless backtrace and sentry have Electron-specific handling for this stuff? 😰

Reply:
So it's a thing we just invented? 😆

Reply:
Just checked, it looks like at least Backtrace is fine not accepting this parameter. And further, it looks like the field name is often corrupted, e.g.

![corrupted field name, it should be "_companyName" but is instead "<80>s<E4><B5>k^B"](https://user-images.githubusercontent.com/172800/81234733-31c07780-8fae-11ea-8df4-1dca62d92bf8.png)

Reply:
> So it's a thing we just invented? 😆

Did some digging, looks like it came from 2013 when we used a crash reporter called "Quincy": https://github.com/electron/electron/commit/cd42fcceb986462506a63e344bdb624878d215c0#diff-a5e9fd206dd9f3e720425acc4e920df6R28

Reply:
From the Quincy source, it looks like it was used to customize the crash dialog: https://github.com/bitstadium/QuincyKit/blob/443214387fccae5558e9a660474ec8482f0a8c9d/client/Mac/BWQuincyManager.h#L167